src/plugins/dam/components/image-asset-fields/index.jsx

```diff
@@ -35,6 +35,13 @@ const ImageAssetFields = ({ asset, isEditMode, schemas }) => (
           placeholder="enter display title"
           readOnly={!isEditMode}
         />
+        <Field
+          name="altText"
+          component={TextField}
+          label="Alt Text"
+          placeholder="alternative information for an image if a user for some reason cannot view it"
```

Comment:
@rsumilang "enter blah" seems like the convention, so "enter alt text" would be good